### PR TITLE
update azure installation guide

### PR DESCRIPTION
**Pull request by sudivate**
- Update azure installation guide to point KFDef manifest for v1.1 for Azure 
[https://github.com/kubeflow/manifests/pull/1487] (url)

CC- @aronchick

### CODE REVIEW

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:16]: Line 16: Would you consider adding the official Docker docs too, such as https://docs.docker.com/docker-for-windows/wsl/? A blog post is OK but less official IMHO.

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:17]: Line 17: I think this Docker Hub link would take the user straight to a login page, which is not awesome in terms of UX, especially if they don't have an account yet. Maybe, a page from https://docs.docker.com/docker-hub/ would suffice?

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:56]: Suggestion: add a quick sentence and back ticks to the code block with command lines:

```suggestion

To log into Azure from the command line interface, run the following commands:

    ```
    az login
    az account set --subscription <NAME OR ID OF SUBSCRIPTION>
    ```

```
(Note the code block back ticks)

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:80]: Line 80: Usually, on the Kubeflow site, the **Note:** paragraph doesn't have to be all bold.

```suggestion
**Note**:  If you are using a GPU-based AKS cluster (For example: AGENT_SIZE=Standard_NC6), you also need to [install the NVidia drivers](https://docs.microsoft.com/azure/aks/gpu-cluster#install-nvidia-drivers) on the cluster nodes before you can use GPUs with Kubeflow.
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:60]: Line 62 (59): You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az group create -n <RESOURCE_GROUP_NAME> -l <LOCATION>
+     ```
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:71]: Line 71 (68): You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
+     ```
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:75]: Lines 75-76: Formatting - back ticks:

```suggestion
- `NAME=KubeTestCluster`
- `AGENT_SIZE=Standard_D4s_v3`

```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:146]: Line 146: add a definite article "the":

```suggestion
1. Open the Kubeflow Dashboard
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:148]: Line 148: surround the site name with back ticks instead of <>:

```suggestion
The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit `http://localhost:8080`.
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hey @sudivate , thanks for the update to Kubeflow deployment on Azure.

I've made a few suggestions.

- Line 16: Would you consider adding the official Docker docs too, such as https://docs.docker.com/docker-for-windows/wsl/? A blog post is OK but less official IMHO.

- Line 17: 

  ```
    - For other OS: [Docker Desktop](https://hub.docker.com/?overlay=onboarding)
  ```

  I think this Docker Hub link would take the user straight to a login page, which is not awesome in terms of UX, especially if they don't have an account yet. Maybe, a page from https://docs.docker.com/docker-hub/ would suffice?

-  Lines 55-56: Add a quick sentence and back ticks to the code block with command lines:

```diff
+ To log into Azure from the command line interface, run the following commands:

+     ```
    az login
    az account set --subscription <NAME OR ID OF SUBSCRIPTION>
+     ```
```

- Line 62: You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az group create -n <RESOURCE_GROUP_NAME> -l <LOCATION>
+     ```
```

- Line 71: You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
+     ```
```

- Lines 75-76: Formatting - back ticks:

```
- `NAME=KubeTestCluster`
- `AGENT_SIZE=Standard_D4s_v3`
```

- Line 80: Usually, on the Kubeflow site, the **Note:** paragraph doesn't have to be all bold. Also, "GPU-based".

```
**Note**:  If you are using...
```

- Line 146: add a definite article "the":

```suggestion
1. Open the Kubeflow Dashboard
```

- Line 148: surround the site name with back ticks instead of <>:

```
The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit `http://localhost:8080`.
```

LMKWYT.

Thanks!

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md:62]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:111]: Can you check number of spaces here please:

```suggestion
   binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:151]:
```suggestion
    ```
    kubectl get all -n kubeflow
    ```
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:156]: Perhaps it may sounds better if the step 1 (run the command) is followed by step 2 (open localhost in the browser ).

Note that you should ideally use single back ticks instead of single quotation marks.

```
-    Run the following command and visit 'http://localhost:8080'.
+    Run the following command:

     ```
     kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
     ```

     Next, open `http://localhost:8080` in your browser.
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed!

Comment (8bitmp3) [reply content/en/docs/azure/deploy/install-kubeflow.md]: Line 3: I think we had a similar thing with IBM Cloud's guide. Maybe it would be better to mention that you're deploying KF with Azure CLI or on Azure

```
- description = "Instructions for deploying Kubeflow with the shell"
+ description = "Instructions for deploying Kubeflow on Azure"
```

WDYT?

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:149]:
```suggestion
2. Run this command to check that the resources have been deployed correctly in namespace `kubeflow`:
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (sudivate) [reply content/en/docs/azure/deploy/install-kubeflow.md]: fixed

**Comment by sudivate** [comment]
@8bitmp3  if you see the netflify preview formatting under "Unpack the tarball:" is broken post-deployment , any pointers what's missing here? However local rendering is perfect

**Comment by 8bitmp3** [comment]
Thank you for your efforts @sudivate

I've just submitted a PR that creates a new OWNERS file to help with Azure docs review process https://github.com/kubeflow/website/pull/2199 - it turns out KF Azure hasn't had official OWNERS assigned yet.

@aronchick Let us know what you think about the changes here and in https://github.com/kubeflow/website/pull/2199

Cheers

**Comment by sudivate** [comment]
@8bitmp3  while we get the approvers added for Azure who can be the provisional approver for this PR?

**Comment by sudivate** [comment]
@animeshsingh  can I please get your approval?

**Pull request merged**
